CalibTracker/SiStripHitResolution/README.md

```diff
@@ -0,0 +1,92 @@
+# Hit Resolution Project #
+
+## Explanation for the pair method ## 
+The hit resolution, $`\sigma_{hit}`$, (in centimetres) can be obtained by using the following equation:
+
+$`\sigma_{hit} = \sqrt{\frac{\sigma^{2}_{HitDX-TrackDX} - \sigma^{2}_{TrackDXE}}{2}}`$,
+
+where $`\sigma_{HitDX-TrackDX}`$ is the standard deviation of the gaussian-fitted distribution of the double difference between the predicted and measured hit positions, and $`\sigma_{TrackDXE}`$ is the mean of the distribution of the predicted positions. 
+
+The following criteria are used:
+
+* The charged particle must have traversed through two sufficiently-overlapping detector modules. 
+* Tracks must have transverse momenta greater than 3 GeV ([permalink](https://gitlab.cern.ch/coldham/hitresolutionproject/-/blob/master/src/HitResol.cc#L324)).
```

Comment:
I think it would make more sense to go back to this once this is merged or at least once the review is finished, otherwise it can quickly become obsolete. What do you think?

Reply:
Yes, that's true, I agree